packages/moonstone/Scrollable/ScrollThumb.js

```diff
@@ -1,45 +1,44 @@
 import {ScrollThumb as UiScrollThumb} from '@enact/ui/Scrollable/Scrollbar';
 import PropTypes from 'prop-types';
-import React, {Component} from 'react';
+import React, {useEffect, forwardRef} from 'react';
 
 const nop = () => {};
 
 /**
  * A Moonstone-styled scroll thumb with moonstone behavior
  *
- * @class ScrollThumb
+ * @function ScrollThumb
  * @memberof moonstone/Scrollable
  * @extends ui/Scrollable/ScrollThumb
  * @ui
  * @private
  */
-class ScrollThumb extends Component {
-	static propTypes = /** @lends moonstone/Scrollable.ScrollThumb.prototype */ {
-		/**
-		 * Called when [ScrollThumb]{@link moonstone/Scrollable.ScrollThumb} is updated.
-		 *
-		 * @type {Function}
-		 * @private
-		 */
-		cbAlertThumb: PropTypes.func
-	}
-
-	static defaultProps = {
-		cbAlertThumb: nop
-	}
-
-	componentDidUpdate () {
-		this.props.cbAlertThumb();
-	}
-
-	render () {
-		const props = Object.assign({}, this.props);
-
-		delete props.cbAlertThumb;
-
-		return <UiScrollThumb {...props} />;
-	}
-}
+const ScrollThumb = forwardRef((props, ref) => {
```

Comment:
What about `{cbAlertThumb, ...rest}` instead of the `props` above? Then we could remove the following two lines.

Reply:
Oh, right.